src/js/disability-benefits/components/ClaimsListItem.jsx

```diff
@@ -20,7 +20,8 @@ export default function ClaimsListItem({ claim }) {
           ? <p><i className="fa fa-exclamation-triangle"></i>Items need your attention</p>
           : null}
       </div>
-      <p>Last update: {moment(claim.attributes.updatedAt).format('MMM D, YYYY')}</p>
+      {claim.attributes.phaseChangeDate &&
```

Comment:
Yeah. It's sometimes awkward to use `if` in JSX components because it's not an expression in Javascript. So there are two ways people do conditionals:

`{someCondition ? <Component/> : null}`

or 

`{someCondition && <Component/>}`

The later is more concise but has some pitfalls. I usually use the former, but seemed alright here.